### PR TITLE
SSCS-3198 date picker accessibility

### PR DESCRIPTION
**Pull request by bendiggle**
- Increased contrast on day of the month
- Added keyboard navigation
- Added aria attributes

The bootstrap datepicker doesn't support keyboard navigation when used in embedded mode. Therefore I had to kinda hammer it in, which results in a slightly hackish appearance.

### CODE REVIEW

File: assets/js/date-picker/date-picker.js [hunk 1]
```diff
@@ -38,25 +116,46 @@ const datePicker = {
       maxViewMode: 0,
       datesDisabled,
       beforeShowDay: date => datePickerUtils.displayFirstOfMonth(date)
-    }).on('changeDate', event => {
-      datePicker.changeDateHandler(event);
-    });
+    }).on('changeDate', event => datePicker.changeDateHandler(event))
+      .on('destroy', event => console.info.bind(this, 'vabffabma'))
```
Comment (bendiggle) [inline assets/js/date-picker/date-picker.js]: @barbaracassani I don't think you meant to leave this here 😄

Comment (barbaracassani) [reply assets/js/date-picker/date-picker.js]: oops 😊

**Comment by bendiggle** [comment]
I think this is to do with the setting the aria attributes to the cell but when it's the first of the month it should look like this: 
screen shot 2018-05-29 at 17 39 20

It currently looks like this:
screen shot 2018-05-29 at 17 38 38

**Comment by bendiggle** [comment]
LGTM!

**Pull request merged**